packages/opentelemetry-instrumentation-openai/opentelemetry/instrumentation/openai/v1/responses_wrappers.py

```diff
@@ -453,6 +495,9 @@ def responses_get_or_create_wrapper(tracer: Tracer, wrapped, instance, args, kwa
         return wrapped(*args, **kwargs)
     start_time = time.time_ns()
 
+    # Normalize kwargs to remove OpenAI sentinel values for safe attribute access
+    safe_kwargs = _sanitize_sentinel_values(kwargs)
```

Comment:
neat: I would change the naming to: `non_sentinel_kwargs` since it's more explicit and self-explanatory imo, and the kwargs still contains Nones which makes it not safe...